source/about-mongosync.txt

```diff
@@ -162,13 +162,15 @@ sync job.
 Considerations for Continuous Sync
 ----------------------------------
 
-Avoid using ``mongosync`` for disaster recovery for the following reasons:
+You can periodically run ``mongosync`` against source and destination clusters 
+in continuous synchronization. Using ``mongosync`` for continuous sync has the 
```

Comment:
wordsmithing nit: instead of the "Using" sentence, we can say "Please bear the following considerations in mind when using mongosync for continuous synchronization:"

Reply:
Will adjust this to align with our docs' style and tone.